.github/workflows/third_party.yml

```diff
@@ -60,19 +60,18 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: typing-extensions-latest
-      - name: Setup pdm for pydantic tests
-        uses: pdm-project/setup-pdm@v4
-        with:
-          python-version: ${{ matrix.python-version }}
-          allow-python-prereleases: true
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        run: uv python install ${{ matrix.python-version }}
       - name: Add local version of typing_extensions as a dependency
-        run: pdm add ./typing-extensions-latest
+        run: uv add --editable ./typing-extensions-latest
       - name: Install pydantic test dependencies
-        run: pdm install -G testing -G email
+        run: uv sync --group testing --group dev
```

Comment:
note: the email dependency group was replaced by the dev group ([relevant commit from the linked PR](https://github.com/pydantic/pydantic/pull/10727/commits/c579e79eafe83bfbaac37ff682db7f155942f1ac))